gui/wxpython/gui_core/pyedit.py

```diff
@@ -515,14 +515,15 @@ def OpenRecentFile(self, path, file_exists, file_history):
                 ),
                 parent=self.guiparent,
             )
-        else:
-            if self.CanReplaceContent(by_message="file"):
-                self.filename = path
-                content = self._openFile(file_path=path)
-                if content:
-                    self.body.SetText(content)
-                    file_history.AddFileToHistory(filename=path)  # move up the list
-                    self.tempfile = False
+            return
```

Comment:
Is the return needed (or reached at all after _GError_ )?

Reply:
I wasn't sure enough when reading the code for GError, so I applied the early exit instead of collapsing, as it was suggesting that usage

Reply:
In case GError changes, just to make sure I'm keeping the return